rootfs/usr/share/www/index.html

```diff
@@ -257,6 +257,15 @@ <h3>While waiting, some suggestions:</h3>
         log.showFull = !log.showFull
         fetchLogs()
       });
+
+      document.addEventListener('DOMContentLoaded', (event) => {
+        if (location.search.indexOf("client_id=http://home-assistant.io/android") !== -1 ||
+        location.search.indexOf("client_id=http://home-assistant.io/iOS") !== -1) {
+          // Hide app links if visited from an app
+          const logElement = document.querySelector("#app-links")
+          logElement.remove()
```

Comment:
So we don't have to debate about var name 😄 

```suggestion
          document.querySelector("#app-links").remove()
```